### ISSUE TITLE
Color Contrast Too Low for Selected Navigation Items

### ISSUE THREAD

**Issue by aardrian** [smashingmagazine/redesign#186]
**- Do you want to request a *feature* or report a *bug*?**
Bug

**- What is the current behavior?**
The very dark red (#58120e) of navigation text that denotes the current section on the orange field (#e33a2c) in the navigation has a contrast ratio of 3.25:1.

**- If the current behavior is a bug, please provide the steps to reproduce.**
View the page in any browser.

**- What is the expected behavior?**
The contrast between the text and its background should be 4.5:1 for normal text and 3:1 for large text to meet WCAG 2.0 item 1.4.3 (AA). The larger text in the navigation item passes because it is large (though it would benefit from more contrast), but the smaller text does not. More information on the success criterion along with some guidelines can be found at https://www.w3.org/WAI/WCAG20/quickref/#visual-audio-contrast-contrast

**- Please mention your operating system version and the version of your browser**
Windows, using Firefox, Chrome, Edge, IE. Android, using Chrome, Firefox. Versions are irrelevant in this case. Do not have Mac / iOS device handy, but expect it looks the same.

Attached are screen shots of the example navigation item showing the issue as well as the output of the [WebAIM Color Contrast Checker](http://webaim.org/resources/contrastchecker/) showing the results:
![2017-03-24 2](https://cloud.githubusercontent.com/assets/1376607/24289553/2b57d90c-1058-11e7-8849-6beb709cd544.png)
![2017-03-24 3](https://cloud.githubusercontent.com/assets/1376607/24289563/32dc3d62-1058-11e7-80d7-797bdfeda69f.png)

**Comment by dstorey** [comment]
this is a dupe of #105

**Comment by aardrian** [comment]
@dstorey Different color values and parts of the page. Can they be merged?
